### ISSUE TITLE
Build status notifications.

### ISSUE THREAD

**Issue by guglez** [assistify/rcapps-gitlab#16]
Hi,

your documentation states this:
> This app connects GitLab with Assistify. Users are notified directly in the Assistify chat about code changes and the current build status. 

However when I test webhook with "job" or "pipeline" event I receive this:
`Hook executed successfully but returned HTTP 500 Unknown GitLab event 'undefined'`

Could you please clarify how to monitor build status?
What kind of events are supported today? Push notifications are working but others don't.

I'm using app version 0.1.0 from rocketchat marketplace.

**Comment by jschirrmacher** [comment]
You're right, the current version 0.1.0 only support push events on gitlab currently. Other events are not yet implemented.

**Comment by jschirrmacher** [comment]
Other GitLab events will be part of version 1.0.1 which will be available in the marketplace soon.
